devices/cpu/registers.h

```diff
@@ -39,6 +41,25 @@ typedef struct Status_reg {
   uint16_t reserved : 7;   // Reserved bits
 } __attribute__((packed)) Status_reg;
 
+// Structure describing call information //
+typedef struct CallEntry {
+  uint16_t targetPc; // Target call PC
+  uint16_t returnPc; // Return PC (one instruction after the call)
+  uint16_t sp;       // SP value at the time of the call
+} CallEntry;
```

Comment:
it might be just me, but `Entry` sounds strange in this context, I'd go with something like "just" `Call` or `CallTrace`, then next structure could be `CallTraces` as data structure, because `Tracer` suggest "active module that does something"

Reply:
Call is misleading. I think CallEntry is better. Still, CallTraceEntry seems also ok.